shaders/src/common_lighting.fs

```diff
@@ -71,6 +75,64 @@ float computeMicroShadowing(float NoL, float visibility) {
     return microShadow * microShadow;
 }
 
+/**
+ * IBL utilities
+ */
+
+vec2 IntersectAABB(vec3 rayOrigin, vec3 rayDir, vec3 boxMin, vec3 boxMax) {
```

Comment:
All these probably need to be `highp`. Also please use the `const` decorator where possible.

Reply:
Added these in 6dc6210f556cee2f8ce36cfd404588c5ddb3fa2d . Shall I `highp` up all local variables in these functions too? I'm less worried about the spherical IBLs as the intersection computations are normalized to [-1,1]^3 but the box IBL's computation domain can be arbitrarily large. Although if you are OK with me using another 3 floats in the UBO to supply the reciprocal half extents, then I can normalize there too. Does this sound reasonable?